ogcapi-features/README.md

```diff
@@ -110,6 +112,85 @@ STAC API.
 Implementing OAFeat enables a wider range of clients to access the API's STAC Item objects, as it is a more widely implemented
 protocol than STAC. 
 
+***Todo*** -- allowing urls like `/collections/landsat-8-l1/151/018` where `landsat-8-l1/151/018` is the collectionId. 
+
+## Item Pagination
+
+OAFeat supports paging through hypermedia links for the Items resource 
+(`/collections/{collectionId}/items`). Since only GET requests are allowed for this endpoint, 
+it is possible
+to provide a Link with `rel` type `next` and the href of the full URL of the next page of results.
+This link href must contain any URL parameters that are necessary 
+for the implementation to understand how to provide the next page of results, eg: `page`, `next`,
+`token`, etc. For example, the links array will have an object like:
+
+```json
+    "links": [
+        {
+            "rel": "next",
+            "href": "http://api.cool-sat.com/collections/my_collection/items?page=2"
+        }
+    ]
+```
+
+The href may contain any arbitrary URL parameter, which is implementation-specific:
+
+- `http://api.cool-sat.com/collections/my_collection/items?page=2`
+- `http://api.cool-sat.com/collections/my_collection/items?next=8a35eba9c`
+- `http://api.cool-sat.com/collections/my_collection/items?token=f32890a0bdb09ac3`
+
+Additionally, STAC has extended the Link object to support additional fields that provide header values
+to the client should they be needed for a subsequent request for the next page of results. The use
+of header values for pagination with GET requests is uncommon, so if your implementation does not use them you can
+omit this attribute in the Link. These
+fields are described in detail in the [Item Search](../item-search/README.md#paging) spec.  To avoid returning
+the entire original request in the response, the  `merge` 
+property can be specified. This indicates that the client must send the same request headers that were sent in
+the original 
+request, but with the specified headers values merged in. This allows servers to indicate what needs to change 
+to get to the next page without mirroring the entire request back to the client. 
+
+See the [paging examples](../item-search/examples.md#paging-examples) for additional insight.
+
+## Collection Pagination
```

Comment:
This section is mentioned in the children conformance class (collections + catalogs) while this section is just pagination for collections, which is confusing to the reader.  Given that pagination over collections/catalogs is identical, I think this section should include catalog pagination.

Reply:
Moreover, paging in general is the same across catalogs, collections, and items is it not?

Reply:
I hope it is! Otherwise, we should unify it.

Reply:
All of the paginations are the same. This section shouldn't mention catalog pagination because it's specifically talking about pagination over the `/collections` endpoint from OAFeat.

This is also outdated now, as it was included b/c I'd branched this PR from another that had the pagination changes in it, which have since been merged.

I do agree about the general point that we have the same pagination description in 5 different places, and it would be nice to consolidate that. One option would be to define the pagination scheme in Core (even through it wouldn't be used) and then reference it in all of the other places.  Filed https://github.com/radiantearth/stac-api-spec/issues/236